frontend/packages/git-service/src/services/bitbucket-service.ts

```diff
@@ -118,9 +134,14 @@ export class BitbucketService extends BaseService {
   };
 
   getFileContent = async (path: string): Promise<string | null> => {
-    const url = `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}/${path}`;
+    const url = this.isServer
+      ? `${this.baseURL}/projects/${this.metadata.owner}/repos/${this.metadata.repoName}/browse/${path}?at=${this.metadata.defaultBranch}`
+      : `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}/${path}`;
```

Comment:
Also here the bitbucket server throws an error if we fetch //package.json: We can remove the starting slash here or somewhere else, that's up to you. :)

As mentioned in slack, we can also use the /raw/ API here to fetch the complete file instead of a paginated, maybe incomplete, version:

```suggestion
    const url = this.isServer
      ? `${this.baseURL}/projects/${this.metadata.owner}/repos/${this.metadata.repoName}/raw${path}?at=${this.metadata.defaultBranch}`
      : `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}/${path}`;
```